src/main/java/teammates/logic/core/FeedbackSessionsLogic.java

```diff
@@ -50,7 +50,41 @@
 import teammates.storage.api.StudentsDb;
 
 public class FeedbackSessionsLogic {
-
+    
+    private static final String ASSUMPTION_FAIL_DELETE_INSTRUCTOR = "Fail to delete instructor respondant for ";
+    private static final String ASSUMPTION_FAIL_RESPONSE_ORIGIN = "Client did not indicate the origin of the response(s)";
+    private static final String ERROR_NUMBER_OF_RESPONSES_EXCEEDS_RANGE = "Number of responses exceeds the limited range";
+    private static final String ERROR_SENDING_EMAILS = "Error while sending emails :";
+    private static final String ERROR_SESSION_UNPUBLISH_ALREADY = "Session is already unpublished.";
+    private static final String ERROR_SESSION_UNPUBLISH_NON_EXISTENT = "Trying to unpublish a non-existent session.";
+    private static final String ERROR_SESSION_UNPUBLISH_PRIVATE = "Private session can't be unpublished.";
+    private static final String ERROR_SESSION_PUBLISH_ALREADY = "Session is already published.";
+    private static final String ERROR_SESSION_PUBLISH_NON_EXISTENT = "Trying to publish a non-existent session.";
```

Comment:
This can actually be phrased in the same format as other "Trying to %s a non-existent feedback session: %s/%s" 
~~~
private static final String ERROR_PUBLISH_NON_EXISTENT_FS =
        String.format(ERROR_NON_EXISTENT_FS_STRING_FORMAT, "publish");
~~~

(Also see suggestion below about removing the `ACTION_*`)